include/si_vars.h

```diff
@@ -0,0 +1,8 @@
+#ifndef _SI_VARS_H_
+#define _SI_VARS_H
+
+#define SI_VAR_0 0xFE363C80
+
+// TODO
+
+#endif
```

Comment:
Newline pls

Reply:
I'm not sure how I feel about this file. What do you imagine going here?

Reply:
I expect we'll add constants for each of the special values that `get_variable` and friends dereference. i.e. `SI_VAR_0` is equivalent to Star Rod's `*Var[0]`.

> we maybe should just shorten it to SI_ instead of SI_STATUS_
:+1: to that. They could go in this file. Might be worth renaming SI_VAR_ (to maybe just VAR_?) since the vars aren't part of that SI return status "enum".

Reply:
I think this should just be called si.h then, and yeah that makes sense I think, but I'm also not sure what these vars actually represent

Reply:
> I'm also not sure what these vars actually represent

`get_variable` and related functions resolve `bytecode`s. Take the following script, for example:
```py
Set *Var[0] 0             # SET 00000002 SI_VAR_0 0
Set *Var[0] *Var[0]       # SET 00000002 SI_VAR_0 SI_VAR_0
```
Notice that the second operand to `Set` is a union between an immediate int and some special constant values (these are very very low, negative values and are never needed as immediates).
So in this case `get_variable(script, 0)` will return `0` whilst `get_variable(script, SI_VAR_0)` will return whatever the value of `script->varTable[0]` is.

Reply:
From Clover's docs:
![Screenshot_20200815_213514](https://user-images.githubusercontent.com/9429556/90321210-7bbae100-df3f-11ea-8b7a-85dd363948c7.png)